server/test/api/exchange.js

```diff
@@ -5,49 +5,34 @@ var chaiHttp = require('chai-http');
 var expect = chai.expect;
 chai.use(chaiHttp);
 
-// workaround for low node versions
-if (!global.Promise) {
-  var q = require('q');
-  chai.request.addPromises(q.Promise);
-}
-
-// environment variables - disable certificate errors
-process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
-
-// base URL
-var url = 'https://localhost:8080';
-var user = { username : 'superuser', password : 'superuser', project: 1};
+var helpers = require('./helpers');
```

Comment:
This is a good idea, much cleaner :+1: 

As soon as this can be pulled this should be encouraged for integration tests going forward. It can also also be applied to tests written in parallel with this pull request.